### ISSUE TITLE
Template comments highlighting

### ISSUE THREAD

**Issue by husseinalhammad** [lachieh/perch-language-support#1]
Hello,

Inside regular HTML comments `<!-- -->`, Perch tags maintain their highlights, which is the behaviour I expect since the tags actually get rendered this way:

```markup
<!--<perch:label for="email" class="label">Email</perch:label>-->
```

However, inside [template comments](https://docs.grabaperch.com/templates/comments/) `<!--* *-->` I think it makes sense for everything inside the comment, including Perch tags, to be in one colour (i.e. default editor highlighting for comments):

```html
<!--*<perch:label for="email" class="label">Email</perch:label>*-->
```

Thanks!

**Comment by lachieh** [comment]
Thanks for this suggestion! Sorry for the delayed response. I'll get to this as soon as I have a chance. Feel free to submit a PR if you get there before me.

**Comment by lachieh** [comment]
Turns out this was pretty easy. v1.1.0 should be available in the [Marketplace](https://marketplace.visualstudio.com/items?itemName=lachieh.perch-language-support) shortly.

Fixed in #3

**Comment by lachieh** [comment]
I will add that there is a caveat with this. You can't comment out a line (or a block) with the standard kb shortcut because that would require added a base language. This extension just extends the `html` language definition so there's no real way to swap between standard HTML comments and Perch Comments.
